spec/create_index_statement_spec.cr

```diff
@@ -2,10 +2,10 @@ require "./spec_helper"
 
 describe LuckyMigrator::CreateIndexStatement do
   it "generates correct sql" do
-    index_def = LuckyMigrator::CreateIndexStatement.new(:users, column: :email, using: :btree, unique: true)
-    index_def.build.should eq "CREATE UNIQUE INDEX users_email_index ON users USING btree (email);"
+    statement = LuckyMigrator::CreateIndexStatement.new(:users, :email).build
+    statement.should eq "CREATE INDEX users_email_index ON users USING btree (email);"
 
-    index_def = LuckyMigrator::CreateIndexStatement.new(:users, column: :email, using: :btree)
-    index_def.build.should eq "CREATE INDEX users_email_index ON users USING btree (email);"
+    statement = LuckyMigrator::CreateIndexStatement.new(:users, column: :email, using: :btree, unique: true).build
+    statement.should eq "CREATE UNIQUE INDEX users_email_index ON users USING btree (email);"
   end
 end
```

Comment:
If you run `crystal tool format` it should fix this missing newline

Reply:
Ah yeah that's convenient, I should make a habit of running it